payment/endpoints.go

```diff
@@ -0,0 +1,37 @@
+package payment
+
+import (
+	"github.com/go-kit/kit/endpoint"
+	"golang.org/x/net/context"
+	"fmt"
+)
+
+// Endpoints collects the endpoints that comprise the Service.
+type Endpoints struct {
+	AuthoriseEndpoint endpoint.Endpoint
+}
+
+// MakeEndpoints returns an Endpoints structure, where each endpoint is
+// backed by the given service.
+func MakeEndpoints(s Service) Endpoints {
+	return Endpoints{
+		AuthoriseEndpoint:  MakeAuthoriseEndpoint(s),
+	}
+}
+
+// MakeListEndpoint returns an endpoint via the given service.
+func MakeAuthoriseEndpoint(s Service) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (response interface{}, err error) {
+		req := request.(authoriseRequest)
+		fmt.Println("STUB:", req)
```

Comment:
I wanted to keep this so when I add something useful to the request, the code is there to handle it. By removing the req I would have to remove the interface, the parameter, etc. Which I didn't want to do because I wanted it to stay as similar to yours as possible.